SUGGESTIONS.md

```diff
@@ -0,0 +1,17 @@
+# Suggestions
+
+### Project
+Good Structure, short and precise that will enable random explorer to find interest in your work
+
+### README
+- Great work, includes all the necessary things, and links to what ever is required by the project
+
+### INSTALL SCRIPT
+- missing - although could be easier to deploy, think about it...
+- missing - configuring /etc/hosts with radical.local 127.0.0.1
+### K8s
+- why not use image created before hand and saved in registry?
+- why build locally ?
+- why not use single yaml file ?
+- why not use namespace ?
+- what is the point using replicaset if you stick to 1 ?
```

Comment:
i only check it on my vm that it works fixed to 2 replica sets on the deploy file